### PR TITLE
Add models the v2 way

### PR DESCRIPTION
**Pull request by sverhoeven**
Refs #330

### CODE REVIEW

File: docs/_static/flowchart_ewatercycle.excalidraw.svg [hunk 1]
```diff
@@ -12,6 +12,6 @@
...
...1e" stroke-width="1" fill="none"></path></g><g transform="translate(602.9596408341088 1455.062541509576) rotate(0 71.77962818660325 26.71820664742188)"><path d="M113.43 55.82 C123.8 52.64, 133.43 53.73, 145.95 54.5 M114.88 54.85 C123.4 54.25, 133.63 53.85, 144.68 53.9" stroke="#1e1e1e" stroke-width="1" fill="none"></path></g><g transform="translate(602.9596408341088 1455.062541509576) rotate(0 71.77962818660325 26.71820664742188)"><path d="M120.32 36.49 C128.4 39.84, 135.71 47.45, 145.95 54.5 M121.77 35.52 C128 41.09, 136.01 46.93, 144.68 53.9" stroke="#1e1e1e" stroke-width="1" fill="none"></path></g></g><mask id="mask-q6Yk4z02mV9-hb7e8QEXw"><rect x="0" y="0" fill="#fff" width="847.1111111111111" height="1608.4785642868758"></rect><rect x="654.8270420034937" y="1469.270552898226" fill="#000" width="40.41666793823242" height="25" opacity="1"></rect></mask><g transform="translate(654.8270420034937 1469.270552898226) rotate(0 19.9122270172183 12.510195258771887)"><text x="20.20833396911621" y="0" font-family="Virgil, Segoe UI Emoji" font-size="20px" fill="#1e1e1e" text-anchor="middle" style="white-space: pre;" direction="ltr" dominant-baseline="text-before-edge">No...</text></g></svg>
```
Comment (BSchilperoort) [inline docs/_static/flowchart_ewatercycle.excalidraw.svg]: There's a link to https://ns.nl in here now!

**Comment by Peter9192** [comment]
Love the flowchart! Not all clickety-clicks worked on my phone or with the docs preview, but the existence of this functionality is very  promising. 

The one thing that I'm currently missing is the possibility to use a local model. Though it's not the preferred way for plugins, it could still be nice to show it, e.g. as a dashed line, to bypass the two containerization steps during development.

File: docs/_static/flowchart_ewatercycle.excalidraw.svg (hunk 1 above)
Comment (sverhoeven) [reply docs/_static/flowchart_ewatercycle.excalidraw.svg]: doh, link removed. 

Next up fix the rtd build, seems rtd is no longer installing bunch of packages, see https://readthedocs.org/projects/ewatercycle/builds/22129307/

File: docs/_static/flowchart_ewatercycle.excalidraw.svg
Comment (BSchilperoort) [inline docs/_static/flowchart_ewatercycle.excalidraw.svg:1]: I thought we renamed the last bubble to "Now everyone can use your model in eWaterCycle"

**Review by BSchilperoort — APPROVED** [review]
Just one comment on the flowchart, I think you overwrote my change in https://github.com/eWaterCycle/ewatercycle/pull/383/commits/591ea369e7ece3d21b2974dfe402a9e4059351db

Otherwise it looks good to me!

File: docs/_static/flowchart_ewatercycle.excalidraw.svg
Comment (sverhoeven) [reply docs/_static/flowchart_ewatercycle.excalidraw.svg:1]: doh, probably forgot to load svg and continued with was already in https://excalidraw.com/

**Pull request merged**